### PR TITLE
fixed create, reference and resources folders --title-case to sentence-case 

### PR DESCRIPTION
**Pull request by vaish567**
Heyo @dichotommy @guimachiavelli 
Updated folders #960  : 
- `create` 
- `reference`  
- `resources` 
Please go through my pull request and if I'm wrong somewhere feel free to comment, I'll make changes asap.

### CODE REVIEW

File: create/how_to/aws.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ After logging into your [AWS Console](https://aws.amazon.com/console), navigate
 
 In the top-right corner, click on the **Launch instances** button to start the process of configuring your MeiliSearch instance.
 
-### 2. Select 'MeiliSearch' AMI from 'Community AMIs'
+### 2. Select 'Meilisearch' AMI from 'Community AMIs'
```
Comment (vaish567) [inline create/how_to/aws.md:22]: Should make changes here?

File: reference/features/field_properties.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Non-searchable fields are most useful for internal information that's not releva
 Even if you make a field non-searchable, it will remain [stored in the database](#data-storing) and can be made searchable again at a later time.
 :::
 
-### The Searchable Attributes List
+### The Searchable attributes list
```
Comment (dichotommy) [inline reference/features/field_properties.md:22]:
```suggestion
### The searchable attributes list
```

File: reference/features/field_properties.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ In other words, the `searchableAttributes` list serves two purposes:
 
 There are two possible modes for the `searchableAttributes` list.
 
-#### Default: Automatic
+#### Default: automatic
```
Comment (dichotommy) [inline reference/features/field_properties.md:33]:
```suggestion
#### Default: Automatic
```

File: reference/features/language.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ We aim to provide global language support, and your feedback helps us [move clos
 
 If you'd like to learn more about how different languages are processed in MeiliSearch, see our [tokenizer documentation](/reference/under_the_hood/tokenization.md).
 
-## Improving Our Language Support
+## Improving Our language support
```
Comment (dichotommy) [inline reference/features/language.md:12]:
```suggestion
## Improving our language support
```

File: resources/faq.md [hunk 5]
```diff
@@ -34,18 +34,18 @@ To find out more about the relevancy of MeiliSearch, take a look at this detaile
...
 
-## How to know if MeiliSearch perfectly fits my use cases?
+## How to know if meilisearch perfectly fits my use cases?
 
 Since MeiliSearch is an open-source and easy-to-use tool, you can give it a try using your data. Follow this [guide](/learn/getting_started/quick_start.md) to get a quick start!
 
 Besides, we published a [comparison between MeiliSearch and other search engines](/learn/what_is_meilisearch/comparison_to_alternatives.md) with the goal of providing an overview of MeiliSearch alternatives.
 
-## Which languages can MeiliSearch handle?
+## Which languages can meilisearch handle?
```
Comment (dichotommy) [inline resources/faq.md:43]:
```suggestion
## Which languages can MeiliSearch handle?
```

File: create/how_to/aws.md [hunk 6]
```diff
@@ -64,7 +64,7 @@ Tags are used to identify your resources in AWS. **They are not required by Meil
 
 Click on **Next: Configure Security Groups**.
 
-### 7. Security Groups: Networking and connectivity
+### 7. Security groups: networking and connectivity
```
Comment (dichotommy) [inline create/how_to/aws.md:67]:
```suggestion
### 7. Security groups: Networking and connectivity
```
I missed this one!

**Comment by dichotommy** [comment]
Hi @vaish567 , is it clear how to proceed? You can either click `commit suggestion` under each of my suggestions, or incorporate the feedback manually, which takes longer, but leads to a cleaner commit history. I'm always here if you have questions 👋🏻 

We are hoping to merge this PR soon, with your help! 🙂 🔥 🚀

File: create/how_to/search_bar_for_docs.md [hunk 7]
```diff
@@ -179,7 +179,7 @@ Read more about [MeiliSearch authentication](/reference/features/authentication.
 
 :::
 
-### For All Kinds of Documentation
+### For all documentation sites
```
Comment (dichotommy) [inline create/how_to/search_bar_for_docs.md:182]:
```suggestion
### For all kinds of documentation
```

**Review by dichotommy — APPROVED** [review]
Hi @vaish567 , apologies for making changes directly to your branch. We usually avoid doing this, but we needed to merge this PR today so that we could move forward and also preserve your hard work 😉 🙏🏻 

Many thanks for your contribution to MeiliSearch! 🌞

**Pull request merged**